src/clients/api/mutations/bridgeXvs/useBridgeXvs.ts

```diff
@@ -0,0 +1,59 @@
+import { BridgeXvsInput, bridgeXvs, queryClient } from 'clients/api';
+import FunctionKey from 'constants/functionKey';
+import { UseSendTransactionOptions, useSendTransaction } from 'hooks/useSendTransaction';
+import {
+  useGetXVSProxyOFTDestContract,
+  useGetXVSProxyOFTSrcContract,
+  useGetXvsVaultContract,
+} from 'packages/contracts';
+import { useGetToken } from 'packages/tokens';
+import { useChainId } from 'packages/wallet';
+import { callOrThrow } from 'utilities';
+
+type TrimmedBridgeXvsInput = Omit<BridgeXvsInput, 'tokenBridgeContract'>;
+type Options = UseSendTransactionOptions<TrimmedBridgeXvsInput>;
+
+const useBridgeXvs = (options?: Options) => {
+  const { chainId } = useChainId();
+  const xvs = useGetToken({
+    symbol: 'XVS',
+  });
+  const xvsVaultContract = useGetXvsVaultContract({ chainId, passSigner: true });
+  const tokenBridgeContractSrc = useGetXVSProxyOFTSrcContract({ chainId, passSigner: true });
+  const tokenBridgeContractDest = useGetXVSProxyOFTDestContract({ chainId, passSigner: true });
+  const tokenBridgeContract = tokenBridgeContractSrc ?? tokenBridgeContractDest;
```

Comment:
Looking at the logic in the Bridge UI, it seems the idea is that on Ethereum and Sepolia we use the dest contract, while on other chains we use the src contract right?

To prevent confusion and also potential bugs, I think we should be more explicit about this logic:

```suggestion
  const tokenBridgeContract = chainId === ChainId.SEPOLIA || chainId === ChainId.ETHEREUM ? tokenBridgeContractDest : tokenBridgeContractSrc;
```